DigitalOffice/Services/ProjectService/Database/Entities/DbProject.cs

```diff
@@ -13,7 +13,6 @@ public class DbProject
         public Guid DepartmentId { get; set; }
         public bool Deferred { get; set; }
         public bool IsActive { get; set; }
-        public ICollection<DbProjectManagerUser> ManagersUsersIds { get; set; }
         public ICollection<DbProjectWorkerUser> WorkersUsersIds { get; set; }
```

Comment:
New migration?

Reply:
Yeah, will add.

My bad, working in a stressful environment

Reply:
Write which team you will succeed with. I tried through "Add-Migration {Name} -Namespace {Namespace}", but it didn't work out for me.

Reply:
@KostinP you mean "command" not team, Pasha xD
I committed changes and wrote how-to there.
In my case, there were already some migrations in Database/Migrations/, so the standard command worked out.

Reply:
I even stopped looking at what the translator gives me :D